plans/rpminspect/README.md

```diff
@@ -0,0 +1,48 @@
+# Rpminspect
+
+<!-- SPHINX-START -->
```

Comment:
Do you think we will use this with Sphinx?

Both Docusaurus and Hugo (used before) work with Markdown directly and are easier to work with.

Reply:
Of course this is not sphinx specific. It's a placeholder, open to any suggestion to the name format. I personally like sphinx because you can do `inter-sphinx` and display the content of another sphinx project dynamically ([example](https://myst-parser.readthedocs.io/en/latest/syntax/cross-referencing.html#examples)). But these are just cosmetics.

Reply:
Ok, so, we can leave it here since this should be ignored by other tools.

Reply:
I mean, this should be used by Docusaurus, i.e. when you `include` it should include the section after the begin-tag and before the end-tag. That way it can be customized on both ends.

E.g. in this case I've separated the links so that they can be customized to whatever internal format is available (in sphinx, that would be using `inter-sphinx`). Main README.md, would be formatted similarly to include any relevant badges only for the Github side.